sdk/managedserviceidentity/Azure.ResourceManager.ManagedServiceIdentities/api/Azure.ResourceManager.ManagedServiceIdentities.netstandard2.0.cs

```diff
@@ -137,13 +135,6 @@ public partial class UserAssignedIdentityResource : Azure.ResourceManager.ArmRes
 }
 namespace Azure.ResourceManager.ManagedServiceIdentities.Models
 {
-    public partial class IdentityAssociatedResourceData : Azure.ResourceManager.Models.ResourceData
```

Comment:
Need using customization code to keep these 2 classes in the new version to avoid breaking change, even it have been deleted

Reply:
Refer this https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/compute/Azure.ResourceManager.Compute/src/Customize/CloudServiceRoleInstanceResource.cs, just copy the code from old version and mark all methods as `Obsolete` and `EditorBrowsable`

Reply:
added